mumble_backend/manage.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
```

Comment:
```suggestion
#!/usr/bin/env python3
```
`python` usually refers to a Python 2 installation

Reply:
Not anymore. Even Debian [does not ship python2 anymore](https://wiki.debian.org/Python). Besides, this file is auto generated by Django when you first create the project. If you use it in a venv as it should be used, you will use ``python`` as symlinked binary to execute this file anyway.

Reply:
On ubuntu 22.04, even though there is no Python2 installation, `python` is undefined. You have to use  `python3` in order to use Python3.

> If you use it in a venv as it should be used, you will use python as symlinked binary to execute this file anyway.

Ah okay - I have never used venvs (until testing your implementation ^^)